docs/sourcemap-api.asciidoc

```diff
@@ -1,14 +1,12 @@
 [[sourcemap-api]]
 == Sourcemap Upload API
 
-The APM Server exposes an API Endpoint to upload source maps,
-helpful if you are using <<rum, real user monitoring (RUM)>>.
+The APM Server exposes an API endpoint to upload source maps for <<rum, real user monitoring (RUM)>>.
 
 [[sourcemap-endpoint]]
 [float]
-=== Upload Endpoint 
-To upload a source map you need to send a `HTTP POST` request
-with `Content-Type` set to `multipart/form-data` to the APM Server source maps endpoint:
+=== Upload endpoint
+Send a `HTTP POST` request with `multipart/form-data` `Content-Type`  to the source map endpoint:
```

Comment:
minor: extra space after `Content-Type`.

Why not wrap all of this in back ticks? Is something missing here (maybe "and" to connect the two things)? `multipart/form-data` `Content-Type`